src/sanitizer.js

```diff
@@ -320,6 +339,12 @@ export function isValidAttr(tagName, attrName, attrValue) {
     return false;
   }
 
+  if (attrName == 'class' &&
+      attrValue &&
+      /(^|\W)i-amphtml-/i.test(attrValue)) {
+    return false;
+  }
```

Comment:
@jridgewell I'm not sure what you mean. Could you explain? Before these changes, the sanitizer didn't sanitize the `class` attribute and did allow assignment to values that match the regex.